code/script.js

```diff
@@ -5,15 +5,132 @@
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
+const userName = prompt(`What is your name?`)
+const userNameUp = userName[0].toUpperCase() + userName.slice(1).toLowerCase()
+alert(`Hi ${userNameUp} !`)
 
 // Step 2 - Food choice
 // Your code goes here
 
 // Step 3 - Subtype choice
 // Your code goes here
 
+const orderMain = parseInt(prompt(`What type of food would you like to order? 
+Please enter the number of the food. 
+1. Pizza 
+2. Pasta 
+3. Salad`))
+
+console.log ("main order", orderMain, typeof orderMain)
```

Comment:
Please remove the the console.log() around the code. Generally they need to be deleted when the code has been tested and works as meant to